### DISCUSSION TITLE
Who's using ReactFlow

### DISCUSSION THREAD

**Discussion by ambroseus** [xyflow/xyflow#1052] [General]
Guess it would be interesting for the community to collect in one place links/repos/articles related to `react-flow` 
( like this one https://github.com/formium/tsdx/issues/206 )

**Comment by ambroseus** [comment]
Atomos — A New Recoil Visualization Tool Powered by React Flow
https://coleredfearn.medium.com/atomos-a-new-recoil-visualization-tool-powered-by-react-flow-6e0404a16b9a
https://www.getatomos.io
https://github.com/oslabs-beta/atomos

**Comment by moklick** [comment]
Hey @ambroseus 

thanks for the hint. I think we can use the "Show & Tell" section for this https://github.com/wbkd/react-flow/discussions/categories/show-and-tell

I already wrote the Atomos team that it would be cool if they could post it there.
